team.php

```diff
@@ -21,12 +21,7 @@
     <link rel="stylesheet" type="text/css" href="css/countdown.css?version=51" />
```

Comment:
Create an error page independently. Do not make changes in team.php 
If u haven't pulled properly do so.

Reply:
I reformatted the code with shift+alt+f... I don't think I changed any code.